docs/Documentation/Slurm/interactive_jobs.md

```diff
@@ -92,50 +96,57 @@ If your multi-node job needs a GUI that uses X-windows, the least fragile mechan
 ```
 $ salloc --time=20 --account=<handle> --nodes=2
 ...
-[hpc_user@r3i5n13 ~]$ (your compute node r3i5n13)
+[hpc_user@x1008c7s6b1n0 ~]$ (your compute node x1008c7s6b1n0)
 ```
 
 Then from your local workstation:
 
 ```
 $ ssh -Y kestrel.hpc.nrel.gov
 ...
-[hpc_user@el1 ~]$ ssh -Y r3i5n13  #(from login node to reserved compute node)
+[hpc_user@kl1 ~]$ ssh -Y x1008c7s6b1n0  #(from login node to reserved compute node)
 ...
-[hpc_user@r3i5n13 ~]$  #(your compute node r3i5n13, now X11-capable)
-[hpc_user@r3i5n13 ~]$ xterm  #(or another X11 GUI application)
+[hpc_user@x1008c7s6b1n0 ~]$  #(your compute node x1008c7s6b1n0, now X11-capable)
+[hpc_user@x1008c7s6b1n0 ~]$ xterm  #(or another X11 GUI application)
 ```
 
-## Requesting Interactive GPU Nodes
+From a Kestrel-DAV Fastx remote desktop session, you can omit the `ssh -Y kestrel.hpc.nrel.gov` above since your terminal in FastX will already be connected to a DAV (kd#) login node. 
```

Comment:
```suggestion
From a Kestrel-DAV FastX remote desktop session, you can omit the `ssh -Y kestrel.hpc.nrel.gov` above since your terminal in FastX will already be connected to a DAV (kd#) login node. 
```